### PR TITLE
Admin service

### PR DESCRIPTION
**Pull request by asanjabi**
On going process of adding AdminService

### CODE REVIEW

File: AzureSaaSDevKit.sln [hunk 1]
```diff
@@ -25,7 +25,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Saas.LandingSignup.Web", "s
 EndProject
 Project("{151D2E53-A2C4-4D7D-83FE-D05416EBD58E}") = "Saas.LandingSignup.Web.Deployment", "src\Saas.LandingSignup\Saas.LandingSignup.Web.Deployment\Saas.LandingSignup.Web.Deployment.deployproj", "{6A0836ED-483E-4F7E-8248-AC00FB52E778}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Saas.Application.Web", "src\Saas.Application\Saas.Application.Web\Saas.Application.Web.csproj", "{547B0C46-F17D-4DA7-9072-3F242D364236}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Saas.Application.Web", "src\Saas.Application\Saas.Application.Web\Saas.Application.Web.csproj", "{547B0C46-F17D-4DA7-9072-3F242D364236}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Saas.Amin.Service.Tests", "src\Saas.Admin\Saas.Amin.Service.Tests\Saas.Amin.Service.Tests.csproj", "{7AAD364B-DEFB-4C66-AEBA-D0A10DBDE45B}"
```
Comment (landonpierce) [inline AzureSaaSDevKit.sln:30]: Saas.Amin typo

Comment (Codebytes) [reply AzureSaaSDevKit.sln]: I pushed an update to rename folder and projects as well as update the sln

File: src/Saas.Admin/Saas.Admin.Service/Services/IPermissionService.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Saas.Admin.Service.Services
+{
+    public interface IPermissionService
+    {
+        Task<IEnumerable<string>> GetSubsriptionUsersAsync(Guid subscriptionId);
```
Comment (Codebytes) [inline src/Saas.Admin/Saas.Admin.Service/Services/IPermissionService.cs:5]: Another typo

**Review by Codebytes — APPROVED** [review]
Looks good to me, admin service and subscription rename works.  
Lets see if we need to tweak the landing signup app

**Pull request merged**